src/main/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -406,6 +399,11 @@ public void setBaitSetName(final String name) {
             this.metrics.PROBE_SET = name;
         }
 
+        /** Returns the accumulated coverage per target. */
+        public Map<Interval,Coverage> getCoverageByTarget() {
```

Comment:
-Space after comma, 
-Note that the map is still *mutable* since it may change "from within" due to another call to acceptRecord. Could you add a comment to that effect?

otherwise, 👍